### ISSUE TITLE
bug: false positive with quoted-expressions

### ISSUE THREAD

**Issue by jpradelle** [43081j/eslint-plugin-lit#209]
With quoted-expressions defined with options "always", following code is reported as error
```js
html`
  <x-foo attr="${v} bar"></x-foo>
`
```

![image](https://github.com/user-attachments/assets/6612b182-4072-4207-9cd8-b69d171250d4)
